CHANGELOG.md

```diff
@@ -17,7 +17,11 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Bugfixes
 
-* None.
+* Fix a call to `commitWrite(withoutNotifying:)` committing a transaction that
+  would not have triggered a notification incorrectly skipping the next
+  notification.
+* Fix collection notifications missing move information for non-end insertions
```

Comment:
Fix incorrect results and crashes when conflicting object insertions are merged by the synchronization mechanism when there is a collection notification active for that object type.